InteractionBase/src/handlers/GenericHandler.cpp

```diff
@@ -424,6 +425,8 @@ void GenericHandler::keyPressEvent(Visualization::Item *target, QKeyEvent *event
 			// Check if the parent of the node is a list and if so, delete this node
 			if (auto list = DCast<Model::List>(node->parent()))
 			{
+				for (auto view : target->scene()->viewItems()->viewItemsAsList())
```

Comment:
This is definitely the wrong place for doing this. There are many ways nodes can be deleted and we need a more generic mechanism for dealing with this. What's wrong with using `Item::removeFromScene`?